tests/e2e/specs/keplr/keplr-spec.js

```diff
@@ -61,10 +61,8 @@ describe('Keplr', () => {
       });
     });
     it(`should disconnect the wallet from all the connected DAPPs`, () => {
-      cy.switchToExtensionWindow().then(() => {
```

Comment:
just a reminder: we should also create a separate PR to move all other instances where explicitly call `switchToExtensionWindow` in our tests